src/main/java/Eliz/Eliz.java

```diff
@@ -0,0 +1,331 @@
+                + "| |__   |  |     |  |      / /  \n"
+                + "| |__|  |  |     |  |     / /  \n"
+                + "| |__   |  |__   |  |    / /___\n"
+                + "|____|  |_____|  |__|   |______|\n";
+
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Eliz");
+        System.out.println("I am your personal task list recorder! If you need help, type [/help]");
+        System.out.println("What task would you like me to note down for you?");
+    }
+
+    public static void markOrUnmark(String line, ArrayList<Task> tasks) {
+        if (line.contains("unmark")) {
+            System.out.println("OK, I've marked this task as not done yet:");
+            unmarkATask(line, tasks);
+        } else {
+            System.out.println("Nice! I've marked this task as done:");
+            markATask(line, tasks);
+        }
+    }
+
+    public static void displayCommands() {
+        System.out.println("Commands to add tasks: ");
+        System.out.println("todo [TASK_NAME]");
+        System.out.println("event [TASK_NAME]");
+        System.out.println("deadline [TASK_NAME]/[DATE]");
+        System.out.println(" ");
+        System.out.println("Command to display list: ");
+        System.out.println("list");
+        System.out.println(" ");
+        System.out.println("Commands for todo tasks: ");
+        System.out.println("mark [TASK_LIST_NUMBER]");
+        System.out.println("unmark [TASK_LIST_NUMBER]");
+    }
+
+    public static void delete(String line, ArrayList<Task> tasks) {
+        char taskNumChar = line.charAt(line.length() - 1);
+        int taskNumInt = Character.getNumericValue(taskNumChar);
+        String deletedTask = tasks.get(taskNumInt - 1).getTaskType();
+        String deletedTaskDescription = tasks.get(taskNumInt - 1).description;
+        tasks.remove(taskNumInt - 1);
+        System.out.println("Noted I have deleted this task");
+        System.out.println("[" + deletedTask + "]" + "[ ] " + deletedTaskDescription);
+        System.out.println("You have " + tasks.size() + " tasks left in the list.");
+    }
+
+
+    /*To append to file whenever there is a change in tasks*/
+    public static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath,true);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+    /*Deals with the input and calling methods to configure inputs*/
+    public static void getInput(String line, ArrayList<Task> tasks, String filePath) throws ElizException, FileSystemNotFoundException, IOException {
+        String[] breakTaskNames = line.split(" ", 2);
+
+//        if (line.substring(line.length() - 1) == " ") { //in the case that the last character is a space
+//            breakTaskNames[0] = line;
+//            breakTaskNames[1] = null;
+//        } else {
+//            breakTaskNames = line.split(" ", 2);
+//        }
```

Comment:
You could delete unused code blocks and lines across your files to increase overall code quality.